src/renderer/components/log-table.tsx

```diff
@@ -582,26 +582,17 @@ export class LogTable extends React.Component<LogTableProps, LogTableState> {
     rowData: entry,
   }: TableCellProps): JSX.Element | string {
     if (entry && entry.meta) {
-      let icon;
-      // When first loads up without search, originalText will not exist.
-      if (!entry.originalText && entry.message) {
-        icon = isReduxAction(entry.message) ? (
-          <Icon icon="diagram-tree" />
-        ) : (
-          <Icon icon="paperclip" />
-        );
-      } else if (entry.originalText) {
-        icon = isReduxAction(entry.originalText) ? (
-          <Icon icon="diagram-tree" />
-        ) : (
-          <Icon icon="paperclip" />
-        );
-      }
-
+      const icon = isReduxAction(entry.message) ? (
+        <Icon icon="diagram-tree" />
+      ) : (
+        <Icon icon="paperclip" />
+      );
       return (
         <div style={{ display: 'flex', gap: '0.25rem' }}>
           <span title={entry.message}>{icon}</span>
-          <span title={entry.message}>{entry.message}</span>
+          <span title={entry.message}>
+            {entry.highlightMessage ? entry.highlightMessage : entry.message}
```

Comment:
```suggestion
            {entry.highlightMessage ?? entry.message}
```

Use null coalescing operator (`??`) instead: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

And we should refactor this into its own variable since we re-use this logic a couple times in this function.